src/controllers/edgeconnect/version/edgeconnect.go

```diff
@@ -0,0 +1,129 @@
+package version
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/Dynatrace/dynatrace-operator/src/api/status"
+	edgeconnectv1alpha1 "github.com/Dynatrace/dynatrace-operator/src/api/v1alpha1/edgeconnect"
+	"github.com/Dynatrace/dynatrace-operator/src/dockerkeychain"
+	"github.com/Dynatrace/dynatrace-operator/src/registry"
+	"github.com/Dynatrace/dynatrace-operator/src/timeprovider"
+	"github.com/containers/image/v5/docker/reference"
+	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const DefaultMinRequestThreshold = 15 * time.Minute
```

Comment:
why is this here defined?
for the dynakube its defined [here](https://github.com/Dynatrace/dynatrace-operator/blob/c1b3de905ad44a3e82b4204ac6806ce862b51b5d/src/api/v1beta1/dynakube/feature_flags.go#L121)

Reply:
put it into a own config.go file in the edgeconnect package